applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -1,7 +1,132 @@
-.. _power_manager:
+.. _nrf_desktop_power_manager:
 
 Power manager module
 ####################
 
 Use the ``power_manager`` module to reduce the energy consumption of battery-powered devices.
```

Comment:
```suggestion
Use the power manager module to reduce the energy consumption of battery-powered devices.
```
Use the module name, not its code name in the docs (except when discussing code).